pkg/scopes/scopes.go

```diff
@@ -16,9 +16,13 @@ const (
 // GetScopedTopics returns a list of scoped topics for a given application from a Pub/Sub
 // Component properties
 func GetScopedTopics(scope, appID string, metadata map[string]string) []string {
-	topics := []string{}
+	var (
+		existM = map[string]struct{}{}
+		topics = []string{}
+	)
 
 	if val, ok := metadata[scope]; ok && val != "" {
+		val = strings.ReplaceAll(val, " ", "")
```

Comment:
This kind of logic should belong in a utility library at this point. Let's take in this PR but if you could check if we can replace this by an existing library, it would be great.